doc/api/worker_threads.md

```diff
@@ -940,6 +960,8 @@ active handle in the event system. If the worker is already `unref()`ed calling
 [`fs.close()`]: fs.md#fs_fs_close_fd_callback
 [`fs.open()`]: fs.md#fs_fs_open_path_flags_mode_callback
 [`markAsUntransferable()`]: #worker_threads_worker_markasuntransferable_object
+[`'online'` event]: #worker_threads_event_online
```

Comment:
I don't think this one is correctly ordered (because `'` should put it lexically before any letter) but I'm also surprised the linter isn't complaining about it. 
¯\\_(ツ)_/¯

Reply:
Thanks. I'll fix it anyways.